### DISCUSSION TITLE
404 error in production with Apache2

### DISCUSSION THREAD

**Discussion by timvoet** [jnunemaker/flipper#546] [Q&A] [answered]
I know this isn't a Flipper-UI specific issue, but I'm hoping someone with more Flipper experience can help me figure out what's wrong with my apache configuration

```
<IfModule mod_ssl.c>
<VirtualHost *:443>
    ServerName domain.com

    DocumentRoot /var/www/app_directory/current/public

    RailsEnv production

    PassengerRuby /home/user/.rbenv/shims/ruby

    <Directory /var/www/app_directory/current/public>
        AllowOverride all
        Options -MultiViews
        Order allow,deny
        Allow from all
    </Directory>

    ErrorLog ${APACHE_LOG_DIR}/error.log
    CustomLog ${APACHE_LOG_DIR}/access.log combined

    Include /etc/letsencrypt/options-ssl-apache.conf
    SSLCertificateFile fullchain.pem
   SSLCertificateKeyFile privkey.pem
</VirtualHost>
</IfModule>
```
Routes look like this
```
mount Flipper::UI.app(Flipper) => 'flipper'
```

Flipper works great in development, but I get a 404 in production.  No page load, a complete 404

**Comment by jnunemaker** [comment]
Moved this to a discussion. I don't personally use apache anymore so I don't have anything to add but it would be great if anyone in the community had ideas.

**Reply by timvoet** [reply]
It would be awesome if someone had any insights.  and while I am working to move to 1) new versions of everything, 2) nginx.  I would really prefer not having to enable from the console everytime

**Comment by timvoet** [comment, marked as answer]
You can close this. I’ve moved to nginx and it works fine.

**Reply by jnunemaker** [reply]
Glad it is working on nginx. Weird that apache was having a problem.
